tools/frr-reload.py

```diff
@@ -422,7 +422,9 @@ def save_contexts(self, key, lines):
             for line in lines:
                 if line.startswith("ip route ") or line.startswith("ipv6 route "):
                     if "null0" in line:
-                        line = re.sub(r"\s+null0(\s*$)", " Null0", line)
+                        line = re.sub(r"\s+null0(\s*$)", " blackhole", line)
```

Comment:
Does it mean the `frr.conf` will always be drifted from what the user actually has?